assets/kube-apiserver/kube-apiserver-deployment.yaml

```diff
@@ -91,10 +91,14 @@ spec:
           cd /tmp
           mkdir input output
           /usr/bin/cluster-config-operator render --config-output-file config --asset-input-dir /tmp/input --asset-output-dir /tmp/output
++         cp /features/cluster-featuregate.yaml /tmp/input
++         /usr/bin/cluster-config-operator render --config-output-file config --asset-input-dir /tmp/input --asset-output-dir /tmp/output --rendered-manifest-files /tmp/input/cluster-featuregate.yaml
```

Comment:
We should only need one render.  I'm not sure if `cp /features/cluster-featuregate.yaml /tmp/input` is sufficient to satisfy the current render or if we indeed need `--rendered-manifest-files /tmp/input/cluster-featuregate.yaml`.